autogen/convert_structs.py

```
@@ -592,10 +598,13 @@ def build_struct(struct):
             row.append('%s, '       % fimmutable)
             row.append('%s, '       % lot       )
             if size != 1:
-                row.append('%s, '       % size      )
-                row.append('sizeof(%s)' % ftype     )
+                row.append('%s, '         % size )
+                row.append('sizeof(%s), ' % ftype)
+                if field['is_c_array']:
+                    row.append('true')
+                else: row[-1] = row[-1][:-2]
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)